src/main/groovy/life/qbic/service/DummyLocationService.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.service
+
+import life.qbic.StatusMapper
+import life.qbic.datamodel.people.Address
+import life.qbic.datamodel.samples.Location
+import life.qbic.domain.sample.Status
+
+
+class DummyLocationService implements IDummyLocationFactory {
+  @Override
+  Location dummyLocation(Status status, Date arrivalDate) {
+    String name = "dummy location"
```

Comment:
does this need to be added to the database?

Reply:
No I don't think so.